config/jobs/containerd/containerd/OWNERS

```diff
@@ -1,27 +1,10 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
-  - bart0sh
-  - bobbypage
-  - derekwaynecarr
-  - endocrimes
-  - pacoxu
-  - sjenning
-  - SergeyKanzhelev
```

Comment:
Note: https://github.com/kubernetes/k8s.io/issues/7709

Reply:
is it a blocking comment? I am just cleaning up permission to avoid copy/paste errors

Reply:
no, it is not blocking, but owners of these jobs should please see the issue